### PR TITLE
[it] Localization term autoscaling

### LINKED ISSUE #1645: [it] Localizzazione Autoscaling
https://glossary.cncf.io/auto-scaling/

### PR DESCRIPTION
**Pull request by sistella**
### Describe your changes
new term localization: auto-scaling

### Related issue number or link (ex: `resolves #issue-number`)
resolves #1645 

### Checklist before opening this PR (put `x` in the checkboxes)
- [ x ] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [ x ] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/it/auto-scaling.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Questa non elasticità comportava maggiori costi di business e/o perdità di business in caso di disservizi dovuti a un sovraccarico di richieste.
+
+Sfruttando il cloud, la [virtualizzazione](/it/virtualization/), e la [containerizzazione](/it/containerization/) delle applicazioni e delle rispettive dipendenze, 
+le organizzazioni possono sviluppare applicazioni scalabili in relazione alla portata delle richieste degli utenti.
+Possono inoltre monitorare l'utilizzo delle applicazioni e sfruttare la loro scalabilità, offrendo così una esperienza ottimale all'utente. Si consideri ad esempio l'aumento dei fruitori che Netflix deve affrontare il venerdì sera. 
+Con *Autoscaling out* si intende l'aggiunta dinamica di nuove risorse, come ad esempio: l'aumento del numero di server che permettano più streaming video concorrenti, server che potranno poi essere terminati una volta che il flusso si sarà normalizzatomeans dynamically adding more resources: for example, 
+increasing the number of servers allowing for more video streaming and scaling back once consumption has normalized.
```
Comment (ugho16) [inline content/it/auto-scaling.md:20]:
```suggestion
Con *Autoscaling out* si intende l'aggiunta dinamica di nuove risorse, come ad esempio: l'aumento del numero di server che permettano più streaming video concorrenti, server che potranno poi essere terminati una volta che il flusso si sarà normalizzato.
```

File: content/it/auto-scaling.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+Per *autoscaling* si intende la capacità di un sistema di essere [scalabile](/it/scalability/) automaticamente, tipicamente in termini di risorse di computazione. 
+Con un sistema dotato di autoscaling, le risorse sono automaticamente aggiunte al bisogno, così che possano corrispondere alla portata delle richieste degli utenti in quel periodo. 
+Il processo di autoscaling è variabile, ed è configurabile per modificare le risorse di un sistema in base a più metriche, come la memoria o il tempo di computazione.
+I [managed service](/it/managed-service/) sono classicamente associati a funzionalità di autoscaling, dal momento che offrono più opzioni e implementazioni della maggior parte dei sistemi on-premise.
+
+In passato le infrastrutture e le applicazioni erano pensate perchè le loro architetture potessero soddisfare persino i picchi di utilizzo dei sistemi. Questo comportava l'impiego di molte risorse in maniera non elastica, cioè che non cambiavano a seconda del numero di richieste degli utenti.
+Questa non elasticità comportava maggiori costi di business e/o perdità di business in caso di disservizi dovuti a un sovraccarico di richieste.
```
Comment (annalisag-spark) [inline content/it/auto-scaling.md:14]:
```suggestion
Questa mancanza di elasticità comportava maggiori costi di business e/o perdita di business in caso di disservizi dovuti a un sovraccarico di richieste.
```

File: content/it/auto-scaling.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+category: Proprietà
+tags: ["infrastruttura", "", ""]
+---
+
+Per *autoscaling* si intende la capacità di un sistema di essere [scalabile](/it/scalability/) automaticamente, tipicamente in termini di risorse di computazione. 
+Con un sistema dotato di autoscaling, le risorse sono automaticamente aggiunte al bisogno, così che possano corrispondere alla portata delle richieste degli utenti in quel periodo. 
+Il processo di autoscaling è variabile, ed è configurabile per modificare le risorse di un sistema in base a più metriche, come la memoria o il tempo di computazione.
+I [managed service](/it/managed-service/) sono classicamente associati a funzionalità di autoscaling, dal momento che offrono più opzioni e implementazioni della maggior parte dei sistemi on-premise.
+
+In passato le infrastrutture e le applicazioni erano pensate perchè le loro architetture potessero soddisfare persino i picchi di utilizzo dei sistemi. Questo comportava l'impiego di molte risorse in maniera non elastica, cioè che non cambiavano a seconda del numero di richieste degli utenti.
```
Comment (annalisag-spark) [inline content/it/auto-scaling.md:13]:
```suggestion
In passato le infrastrutture e le applicazioni erano pensate perché le loro architetture potessero soddisfare persino i picchi di utilizzo dei sistemi. Questo comportava l'impiego di molte risorse in maniera non elastica, cioè che non variavano accordandosi al numero di richieste degli utenti.
```

File: content/it/auto-scaling.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+Questa non elasticità comportava maggiori costi di business e/o perdità di business in caso di disservizi dovuti a un sovraccarico di richieste.
+
+Sfruttando il cloud, la [virtualizzazione](/it/virtualization/), e la [containerizzazione](/it/containerization/) delle applicazioni e delle rispettive dipendenze, 
+le organizzazioni possono sviluppare applicazioni scalabili in relazione alla portata delle richieste degli utenti.
+Possono inoltre monitorare l'utilizzo delle applicazioni e sfruttare la loro scalabilità, offrendo così una esperienza ottimale all'utente. Si consideri ad esempio l'aumento dei fruitori che Netflix deve affrontare il venerdì sera.
```
Comment (annalisag-spark) [inline content/it/auto-scaling.md:18]:
```suggestion
Possono inoltre monitorare l'utilizzo delle applicazioni e sfruttare la loro scalabilità, offrendo così un'esperienza ottimale all'utente. Si consideri ad esempio l'aumento dei fruitori che Netflix deve affrontare il venerdì sera. 
```

**Review by annalisag-spark — CHANGES_REQUESTED** [review]

**Comment by annalisag-spark** [comment]
@sistella non so se avessi intercettato i miei suggerimenti. vedi tu se li ritieni sensati :)

File: content/it/auto-scaling.md (hunk 4 above)
Comment (annalisag-spark) [reply content/it/auto-scaling.md:18]: @sistella qui ci sono indicazioni pending, appena puoi valutale e possiamo andare avanti, credo :)

**Review by annalisag-spark — APPROVED** [review]

**Pull request merged**